.gitignore

```diff
@@ -6,4 +6,8 @@ __pycache__/
 .env
 venv/
 
-*.log
\ No newline at end of file
+*.log
+
+# ignore the vscode
+.vscode/
```

Comment:
We can keep this workspace config in repo

Reply:
okay,